evidence/doordashcon/0002-doordashcon-08-2024.md

```diff
@@ -0,0 +1,66 @@
+# Evidence-0002: Retention at Rank I
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/08/21)                                                             |
+| **Submitted by**| Jesse Chejieh                                                 |
+
+
+## Member details
+
+- Matrix username: `@doordashcon/:matrix.org`
+- Polkadot address: `12zsKEDVcHpKEWb99iFt3xrTCQQXZMu477nJQsTBBrof5k2h`
+- Current rank: `1`
+- Date of initial induction: `2022/09/29`
+- Date of last report: `2024/05/04`
+- Area(s) of Expertise/Interest: `Frame & Runtime`
+
+
+## Reporting period
+
+- Start date: 2024/05/05
+- End date: 2024/08/24
+
+
+## Evidence
+I believe my contributions over the past period demonstrate my commitment to Polkadot's development and are in accordance with a Rank I member of the fellowship.
+
+I have been actively involve in the development on the polkadot runtime and continued maintainance of the Frame module of the Polkadot-SDK.
+
+Grateful that I have been active in adding new features and collaborated with some higher ranked members of the fellowship, their guidiance has been invaluable to me.
+
+### Features
+Adding the [Secretary collective](https://github.com/polkadot-fellows/runtimes/pull/347) to the Collective System chain.
+
+### Mainteinance
```

Comment:
Are you active in any communication channels?

Reply:
I believe I am active in the Polkadot Technical Fellowship Channel on Matrix.

Reply:
and I am also active in the SE channel on Matrix